src/api/getTrendingArticles.js

```diff
@@ -13,6 +13,7 @@ export const getTrendingArticles = (lang, country) => {
   return cachedFetch(url, data => {
     const page = data.query.pages[0]
     if (page.missing) {
+      sendErrorLog({ message: page.title, url })
```

Comment:
The `message` here needs to be the actual error message we will see in logstash. You can make it something like `Unable to fetch ${page.title} because blah...`. You can even happen `JSON.stringify(data)` at the end to make sure we have the full API response at our disposal to troubeshoot the issue.

Reply:
That's a perfectly great idea right there